BUILD.bazel

```diff
@@ -0,0 +1,407 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Copyright 2020 Julian Amann. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description:
+#   Bazel build file for the ILM OpenEXR libraries.
+
+# Notes:
+# Copied from https://github.com/googlevr/seurat/blob/master/third_party/openexr.BUILD
+# Did some modifications to work with this OpenEXR version: https://github.com/Vertexwahn/openexr
+# (fork from master of OpenEXR around 28. September 2020)
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "-layering_check",
+        "-parse_headers",
+    ],
+)
+
+config_setting(
+    name = "linux_x86_64",
+    values = {"cpu": "k8"},
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "windows_x86_64",
+    values = {"cpu": "x64_windows"},
+    visibility = ["//visibility:public"],
+)
+
+DEFINES = [
+    "DISABLE_GOOGLE_GLOBAL_USING_DECLARATIONS",
```

Comment:
Is this intentional? My understanding is that this is set for the preprocessor when building OpenEXR, but that definition isn't part of OpenEXR's code.

Reply:
Fixed